### ISSUE TITLE
ONNX model for RandomForestClassifier with a single node returns the wrong label

### ISSUE THREAD

**Issue by interesaaat** [onnx/onnxmltools#415]
To reproduce:
```
import numpy as np
from sklearn.ensemble import RandomForestClassifier
from onnxconverter_common.data_types import FloatTensorType
from onnxmltools.convert 

np.random.seed(0)
X = np.random.rand(1, 1)
X = np.array(X, dtype=np.float32)
y = np.random.randint(1, size=1)
model = RandomForestClassifier(n_estimators=5).fit(X, y)
model.fit(X, y)

onnx_ml_model = convert_sklearn(
            model, initial_types=[("input", FloatTensorType([X.shape[0], X.shape[1]]))], target_opset=11)
)

# Get the predictions for the ONNX-ML model
session = ort.InferenceSession(onnx_ml_model.SerializeToString())
output_names = [session.get_outputs()[i].name for i in range(len(session.get_outputs()))]
onnx_ml_pred = [[] for i in range(len(output_names))]
inputs = {session.get_inputs()[0].name: X}
pred = session.run(output_names, inputs)
for i in range(len(output_names)):
    if "label" in output_names[i]:
        onnx_ml_pred[1] = pred[i]
    else:
        onnx_ml_pred[0] = pred[i]

# Get predictions for Sklearn
skl_pred = [[] for i in range(len(output_names))]
skl_pred[0] = model.predict_proba(X)
skl_pred[1] = model.predict(X)

np.testing.assert_allclose(onnx_ml_pred[1], skl_pred[1])  # labels
np.testing.assert_allclose(
    list(map(lambda x: list(x.values()), onnx_ml_pred[0])), skl_pred[0])  # probs
```

Probabilities match, but not the labels (the ONNX model returns label `1` for some reason).
